controllers/metal3.io/preprovisioningimage_controller.go

```diff
@@ -351,9 +353,22 @@ func (r *PreprovisioningImageReconciler) CanStart() bool {
 	return false
 }
 
-func (r *PreprovisioningImageReconciler) SetupWithManager(mgr ctrl.Manager) error {
+func (r *PreprovisioningImageReconciler) SetupWithManager(mgr ctrl.Manager, options controller.Options) error {
```

Comment:
Since the only supported option is concurrency, you could just accept it as an integer.